backend/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/jumpei00/board/backend/app/application"
+	"github.com/jumpei00/board/backend/app/infrastructure"
+	"github.com/jumpei00/board/backend/app/interfaces"
+)
+
+func main() {
+	router := gin.Default()
+
+	group := router.Group("/api")
+
+	// DB
+	threadDB := infrastructure.NewThreadDB()
+
+	// application
+	threadApp := application.NewThreadApplication(threadDB)
+
+	// handler
+	threadHandler := interfaces.NewThreadHandler(threadApp)
+
+	// router setup
+	threadHandler.SetupRouter(group)
+
+	router.Run(":8080")
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `router.Run` is not checked (errcheck)